### PR TITLE
Add support for setting `mirror_type` to `none`

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: REFERENCE.md [hunk 1]
```diff
@@ -80,23 +80,23 @@ Default value: `'/usr/ports'`
 
 Data type: `Array[String]`
 
-Array of options to write to pkg.conf(5)
+Extra ptions to write to pkg.conf(5)
```
Comment (fraenki) [inline REFERENCE.md:83]: s/ptions/options/

File: REFERENCE.md [hunk 2]
```diff
@@ -134,63 +134,63 @@ The following parameters are available in the `pkgng::repo` defined type:
...
+Transport used to reach the repository
 
 Default value: `'http'`
 
 ##### <a name="-pkgng--repo--mirror_type"></a>`mirror_type`
 
 Data type: `Pkgng::Mirror_type`
 
-String of srv or http
+Mirror typo of the repository
```
Comment (fraenki) [inline REFERENCE.md:153]: s/typo/type/

File: manifests/init.pp [hunk 3]
```diff
@@ -16,9 +16,9 @@
 # @param pkg_dbdir Full path to database directory for pkg(8)
 # @param pkg_cachedir Full path to cache directory for pkg(8)
 # @param portsdir Full path to ports directory
-# @param options Array of options to write to pkg.conf(5)
-# @param purge_repos_d Boolean when true removes unmanaged repos
-# @param repos Hash of resources to pass to create_resources()
+# @param options Extra ptions to write to pkg.conf(5)
```
Comment (fraenki) [inline manifests/init.pp:19]: s/ptions/options/

File: manifests/repo.pp [hunk 4]
```diff
@@ -6,14 +6,14 @@
...
-# @param protocol String Transport for repo, from http, https, ftp, file or ssh
-# @param mirror_type String of srv or http
-# @param repopath Path at packagehost url server beginning with /
-# @param enabled Boolean to enable or disable the repository
-# @param priority Interger specifying the order of precedence in multi-repo
-# @param pubkey String containing the full path to the public key to use
-# @param fingerprints String containingthe full path to file containing valid fingerprints, see pkg.conf(8)
+# @param packagehost Hostname of the repository
+# @param protocol Transport used to reach the repository
+# @param mirror_type Mirror typo of the repository
```
Comment (fraenki) [inline manifests/repo.pp:11]: s/typo/type/

**Comment by smortex** [comment]
@fraenki fixed, thanks!

**Review by fraenki — APPROVED** [review]

**Pull request merged**